test/test_export.py

```diff
@@ -150,5 +150,4 @@ def test_autoround_format(self):
     #     inputs = tokenizer(text, return_tensors="pt").to(model.device)
     #     print(tokenizer.decode(model.generate(**inputs, max_new_tokens=50)[0]))
     #     shutil.rmtree("./saved", ignore_errors=True)
```

Comment:
add unittest to export and run generation

Reply:
unittest would fail because autoawq is not on the required package list and an error would raise.

Reply:
add the ut and skip the test if awq has not been installed, we could test it locally and refine the ut system in the future.